client/public/index.html

```diff
@@ -15,6 +15,10 @@
 
     <title>React App</title>
     <link rel="stylesheet" href="styles.css" />
+    <link
```

Comment:
change this whole `<link />` tag with
`<link href="https://fonts.googleapis.com/css2?family=Raleway:wght@600&display=swap" rel="stylesheet">`